qendpoint-core/src/main/java/com/the_qa_company/qendpoint/core/compact/sequence/SequenceInt32.java

```diff
@@ -40,6 +40,7 @@ public class SequenceInt32 implements DynamicSequence {
 	/** The array that holds the objects **/
 	int[] data;
 	int numelements;
+	private final long[] prevFound = new long[16384];
```

Comment:
The size of the prevFound arrays are hard coded. It should instead be calculated based on the size of the LongArray and the bucket size.